### DISCUSSION TITLE
Delete all local images

### DISCUSSION THREAD

**Discussion by pavellety** [cirruslabs/tart#375] [Q&A] [answered]
Would it be possible to add option to tart delete something like '--all' which would delete all images at once?

Thanks,
Pavel

**Comment by edigaryev** [comment, marked as answer]
You can use `tart list --source local --quiet` to accomplish that.

**Reply by DominikMoreira** [reply]
Hi!

I am using the command `tart list -q | xargs tart delete `to delete VMs. However, when I have a VM called, let's say VM1, the tart list command shows both VM1 and its full reference (e.g., ghcr.io/cirruslabs/VM1@sha256:a5dsf15sdaf). When I execute tart delete VM1, both VM1 and the reference are being deleted. Causing an error that the specified VM does not exist. 

How do you recommend solving this? I am thinking of adding a `grep` to the command, so it would filter out the references. Something like this `'tart list -q | grep -v "ghcr.io" | xargs tart delete`. But I do not like this solution so much. 

Is it intended, that the reference is being deleted from a `tart delete` command?

**Reply by edigaryev** [reply]
Hi @DominikWunderlich,

My bad, I've edited my to include a proper solution.

You need to additionally pass `--source local` to `tart list`.
